models/noise_generator.h

```diff
@@ -96,6 +96,10 @@ frequency double - Frequency of sine modulation in Hz
 
    To obtain comparable results for different values of dt, you must
    adapt std.
+ - As The noise generator provides a different current for each of its targets,
```

Comment:
Ah... I missed that typo! ... I can try my hand at quickly writing a quick test case. I shall commit the typo change and the test case in a couple of hours. Keen to have this done and dusted today itself.

For the test, I presume a simple check to ensure that the current vectors are being populated should suffice? With the length of the current vector being one less than the length of the membrane potential vector. Does that seem sufficient?